### DISCUSSION TITLE
Plugin filter searching not working - NetboxModelFilterSet

### DISCUSSION THREAD

**Discussion by cslingerland** [netbox-community/netbox#9923] [Q&A] [answered]
I'm not able to search my models (yes, they are using the NetboxModel class) using the FilterSets as defined in the documentation. No matter what I type in the search bar in the generic.ObjectListView for this model, it is not doing any filtering. What am I missing?

Here's my View:
```
class VendorListView(generic.ObjectListView):
    queryset = Vendor.objects.all()
    filterset = VendorFilterSet
    table = VendorTable
```

And here's the FilterSet:
```
class VendorFilterSet(NetBoxModelFilterSet):
    class Meta:
        model = Vendor
        fields = ('tenant', 'vendor', 'ip', 'dc', 'notes')
```

**Comment by jeremystretch** [comment, marked as answer]
To enable general searching, you need to override NetBoxModelFilterSet's `search()` method and define the fields to search on. By default this returns the entire queryset. Here's an example from the Site model:

https://github.com/netbox-community/netbox/blob/a2e84dd279549c7284184ff98da4a1ce23c646c9/netbox/dcim/filtersets.py#L151-L166

**Reply by cslingerland** [reply]
Thank you! That was it
